src/DatePicker/DatePickerDialog.js

```diff
@@ -73,129 +77,98 @@ class DatePickerDialog extends React.Component {
     this.dismiss();
   };
 
-  handleTouchTapOK = () => {
+  handleTouchTapOk = () => {
     if (this.props.onAccept && !this.refs.calendar.isSelectedDateDisabled()) {
       this.props.onAccept(this.refs.calendar.getSelectedDate());
     }
 
     this.dismiss();
   };
 
-  handleKeyUp = (event) => {
+  handleKeyUp = (event) => { // FIXME: Unused function
     switch (keycode(event)) {
       case 'enter':
-        this.handleTouchTapOK();
+        this.handleTouchTapOk();
         break;
     }
   };
 
   render() {
     const {
+      // autoOk, FIXME: autoOk not working
       DateTimeFormat,
       cancelLabel,
       container,
+      disableYearSelection,
       initialDate,
       firstDayOfWeek,
       locale,
+      maxDate,
+      minDate,
+      mode,
       okLabel,
       onAccept, // eslint-disable-line no-unused-vars
+      shouldDisableDate,
       style, // eslint-disable-line no-unused-vars
       wordings,
-      minDate,
-      maxDate,
-      shouldDisableDate,
-      mode,
-      disableYearSelection,
       ...other,
     } = this.props;
 
     const {open} = this.state;
 
-    const {
-      datePicker: {
-        calendarTextColor,
-      },
-    } = this.context.muiTheme;
-
     const styles = {
-      root: {
-        fontSize: 14,
-        color: calendarTextColor,
-      },
-
       dialogContent: {
-        width: mode === 'landscape' ? 480 : 320,
+        width: mode === 'landscape' ? 479 : 310,
       },
-
       dialogBodyContent: {
-        padding: 0,
-      },
-
-      actions: {
-        marginRight: 8,
+        paddingLeft: 0,
+        paddingBottom: 0,
+        minHeight: mode === 'landscape' ? 330 : 434,
+        minWidth: mode === 'landscape' ? 479 : 310,
       },
     };
 
-    const actions = [
-      <FlatButton
-        key={0}
-        label={wordings ? wordings.cancel : cancelLabel}
-        primary={true}
-        style={styles.actions}
-        onTouchTap={this.handleTouchTapCancel}
-      />,
-    ];
-
-    if (!this.props.autoOk) {
-      actions.push(
-        <FlatButton
-          key={1}
-          label={wordings ? wordings.ok : okLabel}
-          primary={true}
-          disabled={this.refs.calendar !== undefined && this.refs.calendar.isSelectedDateDisabled()}
-          style={styles.actions}
-          onTouchTap={this.handleTouchTapOK}
-        />
-      );
-    }
-
-    // will change later when Popover is available.
-    const Container = (container === 'inline' ? DatePickerInline : Dialog);
+    const Container = (container === 'inline' ? Popover : Dialog);
     return (
-      <Container
-        {...other}
-        ref="dialog"
-        style={styles.root}
-        contentStyle={styles.dialogContent}
-        bodyStyle={styles.dialogBodyContent}
-        actions={actions}
-        repositionOnUpdate={false}
-        open={open}
-        onRequestClose={this.handleRequestClose}
-      >
-        {open &&
+      <div {...other} ref="root">
+        <Container
```

Comment:
cc @newoga 

This bit feels wrong to me. We're passing all the props here for 2 completely different components.

I don't think this should be necessary. If we're making users put `Menu` inside a `Popover`, why are we providing it internally inside DatePicker? (especially when arguably `Menu` **is** popover according to the spec, and DatePicker should just be placed inside a menu).